server/services/datastoreSearchService.ts

```diff
@@ -0,0 +1,28 @@
+import logger from '../../logger'
+import getSanitisedError from '../sanitisedError'
+import { Order } from '../interfaces/order'
+import DatastoreClient from '../data/datastoreClient'
+import { HmppsAuthClient, RestClientBuilder } from '../data'
+
+export default class DatastoreSearchService {
+  private readonly datastoreClient: DatastoreClient
+
+  constructor(
+    private readonly datastoreClientFactory: RestClientBuilder<DatastoreClient>,
+    private readonly hmppsAuthClient: HmppsAuthClient,
+  ) {
+    this.datastoreClient = this.datastoreClientFactory('uninitialised')
+  }
+
+  async getOrders(criteria: Order): Promise<Order[]> {
```

Comment:
I think this would be better as `SearchForOrders()`, matching the client method.

Reply:
Missed this one - will add now.